CMakeLists.txt

```diff
@@ -42,21 +45,20 @@ target_compile_definitions(mconfig PUBLIC MCONFIG_LIB)
 
 option(mconfig-crypto "MConfig: Build with encryption functions" OFF)
 if(mconfig-crypto)
+  message("MConfig: using encryption library")
+  target_compile_definitions(mconfig PRIVATE CRYPTED_CONFIG)
```

Comment:
@lkorbel should probably be `ENCRYPTED_CONFIG`.

Also, similar changes will be required for qmake support (.pri file)

Reply:
@sierdzio Thank you, I will rename it.